src/components/Search.js

```diff
@@ -0,0 +1,19 @@
+import React from 'react'
+function Search (props) {
+    return ( 
+        <form className="navbar-form navbar-left" role="search">
+        <div className="input-group mb-3">
+            <input type="text" className="form-control" placeholder="Enter movie's name" onChange={props.onkeyHandle} />
```

Comment:
please rename onkeyHandle to onKeyHandle

Reply:
Renamed